src/modules/test/commentsCount.test.js

```diff
@@ -0,0 +1,22 @@
+import commentCounter from '../commentscount.js';
+
+global.fetch = jest.fn();
+describe('commentCounter', () => {
+  beforeEach(() => {
+    fetch.mockReset();
+  });
+
+  test('returns the number of comments', async () => {
+    const comment = [{ id: 1 }, { id: 2 }, { id: 3 }];
+    fetch.mockResolvedValue({ json: () => Promise.resolve(comment) });
+    const result = await commentCounter();
+    expect(result.length).toBe(3);
+  });
```

Comment:
- [ ]  In tests of counter modules you are supposed to consider all possible edge cases, for instance, the scenario in which there 0 items on page or 0 comments, you should consider those to all the modules and test them as well to make you project full the requirements. Remember writing good tests make your project more stable.